pkgs/applications/office/paperless-ngx/default.nix

```
@@ -63,6 +65,9 @@ let
 
     buildInputs = [
       pango
+      giflib
+    ] ++ lib.optionals stdenv.isDarwin [
```

```suggestion
    ] ++ lib.optionals stdenv.isDarwin [
      giflib
```
couldn't reproduce the failure on linux